src/docs/command-line.dox

```diff
@@ -28,18 +28,23 @@ Verify the WiredTiger metadata as part of opening the database.
 Run recovery if the underlying database is configured to do so.
 @par \c -r
 Access the database via a readonly connection
+@par \c -S
+Run salvage recovery if the underlying database is configured to do so.
 @par \c -V
 Display WiredTiger version and exit.
 @par \c -v
 Set verbose output.
+@par \c -?
+Print the help message and exit.
 
 Unless otherwise described by a \c wt command, the \c wt tool exits zero
 on success and non-zero on error.
 
-The \c wt tool supports several commands.  If configured in the underlying
-database, some commands will run recovery when opening the database.  If
-the user wants to force recovery on any command, use the \c -R option.
-In general, commands that modify the database or tables will run recovery
+The \c wt tool supports several commands.  You can use the \c -? option
+with any of these commands to print the help message.  If configured in the
+underlying database, some commands will run recovery when opening the database.
+If the user wants to force recovery on any command, use the \c -R or the \c -S
+option. In general, commands that modify the database or tables will run recovery
```

Comment:
I'm nervous about casually suggesting the `'S'` option.  `'R'` recovery is normal, it should never lose data.  `'S'` salvage can be done if all else fails, it's an emergency option.  It can restore data that was previously deleted (and committed!) and remove inserted, committed data.  I'd recommend taking out `-S` from this sentence - we don't describe every option here, and salvage can be found elsewhere by those in need.

(See https://github.com/wiredtiger/wiredtiger/blob/bee74c39df9a3ba11449b839057ecb86be32c935/src/include/wiredtiger.in#L1591)

Reply:
Ok, thanks! I changed the sentence to:

`If the user wants to force recovery on any command, use the \c -R option.`